.gitignore

```diff
@@ -84,3 +84,8 @@ venv.bak/
 
 # In case you place source code in ~/nni/
 /experiments
+src/sdk/pynni/nni/nni
```

Comment:
why add these folders?

Reply:
They are links, and created by dev-install.